lua/guard/filetype.lua

```diff
@@ -1,43 +1,61 @@
 local M = {}
 
+local function get_tool(tool_type, tool_name)
+  if not pcall(require, 'guard-collection.' .. tool_type) then
+    vim.notify(
+      ('[Guard]: "%s": needs nvimdev/guard-collection to access builtin configuration'):format(
+        tool_name
+      ),
+      4
+    )
+    return
+  end
+  local tbl = require('guard-collection.' .. tool_type)[tool_name]
```

Comment:
```suggestion
  local is_success, tool = pcall(require, 'guard-collection.' .. tool_type)
  if not is_success then
    vim.notify(
      ('[Guard]: "%s": needs nvimdev/guard-collection to access builtin configuration'):format(
        tool_name
      ),
      4
    )
    return
  end
  local tbl = tool[tool_name]
```
You don't have to `require` twice